utils/errors/notifier/notifier.go

```diff
@@ -109,21 +128,25 @@ func convToSentry(in errors.ErrorExt) *raven.Stacktrace {
 	return out
 }
 
-func parseRawData(ctx context.Context, rawData ...interface{}) map[string]interface{} {
+func parseRawData(ctx context.Context, rawData ...interface{}) (map[string]interface{}, []interface{}) {
```

Comment:
Use named return parameters here so the method signature is self explanatory. Something like `rawData map[string]interface{}, tags []interface{}`